app/views/accounts/organizations/_teams.html.erb

```diff
@@ -114,10 +114,20 @@
                 <% modal.with_body do %>
                   <%= render partial: "accounts/invitations/edit_form", locals: { email: member.email, member_role: member_role } %>
                 <% end %>
-              </div>
               <% end %>
+              <% end %>
+              <% if can_current_user_remove_member?(member) %>
+              <%= render V2::ButtonComponent.new(
+                scheme: :danger,
+                type: :button,
+                size: :xxs,
+                options: accounts_organization_membership_path(current_organization, member.membership_for(current_organization)),
+                html_options: { method: :delete, data: { turbo_method: :delete, turbo_confirm: "Are you sure you want to remove #{member.email} from the organization?" } }) do |b|
+                b.with_icon("v2/trash.svg")
+              end %>
+              <% end %>
```

Comment:
**todo:** the indentation is off here, we use 2-space indents for erb as well. You can perhaps set your editor to do that. I haven't yet added this to the codebase: https://github.com/Shopify/erb_lint because it had some issues, so you'll have to ensure your editor does it for now.